docs/en/query_language/table_functions/generate.md

```diff
@@ -0,0 +1,38 @@
+# generate
```

Comment:
We have the [template](https://github.com/ClickHouse/ClickHouse/blob/master/docs/dscr-templates/template-function.md) for function descriptions.

Reply:
Actually it is very hard to find if you just try to add new file. We can try to place this templates just near normal .md files, without adding them to documentation.

Reply:
@BayoNet I don't care about templates for functions.